syncer/syncer.go

```diff
@@ -0,0 +1,272 @@
+package syncer
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/helper/progress"
+	"github.com/0xPolygon/polygon-edge/network/event"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+const (
+	LoggerName  = "syncer"
```

Comment:
Why is this exported?

Reply:
Fixed at d75dc06974a15bbf55ba2bc69f8173b95e531db2